content/guides/virtualizing/vmware-fusion.md

```diff
@@ -0,0 +1,104 @@
+There are several methods offered to install Haiku. Among those, only the vmdk and the iso methods are suitable for running Haiku easily on VMware Fusion. Basically these are the two methods that will be shown in this guide. The easiest of all is utilising the vmdk which is already properly configured.
+
+> Running Haiku from a VM image is recommended to avoid any possible problems. Unless it does not work, or you would like to simulate an actual installation, do stick to this method.
```

Comment:
This needs to be an infobox. See the already converted guides or other parts of the site for how to do that.